RELEASE/scripts/autoscend/quests/level_10.ash

```diff
@@ -423,3 +423,14 @@ boolean L10_rainOnThePlains()
 	}
 	return false;
 }
+
+boolean L10_needUmbrella()
+{
```

Comment:
if in Boris or WotSF, we don't need an umbrella because we can't use it (unless an umbrella on a disembodied hand works).

we also can't equip the candy cane sword though (although disembodied hand might work).